src/components/common/ProfilePanel.js

```diff
@@ -208,7 +217,7 @@ const ProfilePanel = function (props) {
     return (
       <form className="panel-edit-container" onSubmit={onNameSubmit}>
         <input
-          autoFocus
+          // autoFocus
```

Comment:
A11y complains about autofocus, but I honestly don't think it's clear that you can now edit if we don't autofocus.